modules/oauth-web-worker/.eslintrc.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
```

Comment:
Shall we extend the existing common lint config.

```suggestion
module.exports = {
    extends: [
        "../../.eslintrc.js"
    ]
};

```